src/utils/index.ts

```diff
@@ -23,3 +24,7 @@ export const logClassNames = (
     console.log(classNames);
   }
 };
+
+export function lazy(arg: TemplateStringsArray) {
```

Comment:
This would only work for non-interpolated tagged templates. The following function would allow interpolations:

```ts
import type { Token } from 'twind';
function lazy(arg: TemplateStringsArray, ...interpolations: Token) {
  return ({ tw }: Context) => tw(arg, ...interpolations);
}
```

Reply:
Good call...I got in a bit of a rush and forgot to account for the interpolations.